src/PostgREST/RangeQuery.hs

```diff
@@ -63,6 +65,9 @@ rangeGeq :: Integer -> NonnegRange
 rangeGeq n =
   Range (BoundaryBelow n) BoundaryAboveAll
 
+allRange :: NonnegRange
+allRange = rangeGeq 0
```

Comment:
This is nice, I like how how it standardizes the rest of the code to say `allRange`.